stream-testkit/src/main/mima-filters/1.1.1.backwards.excludes/pr-1468-override-timeout-stream-testkit.backwards.excludes

```diff
@@ -0,0 +1 @@
+ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.pekko.stream.testkit.scaladsl.StreamTestKit.assertNoChildren")
```

Comment:
When adding mima excludes, please add the reason why it is acceptable. In this case, the reason might be we don't promise binary compatiblity for -testkit artifacts (https://pekko.apache.org/docs/pekko/current/common/binary-compatibility-rules.html), and that this method is `private[testkit]`